backend/src/controllers/team.controller.js

```
@@ -4,6 +4,30 @@ const { internalServerError } = require("../utils/errors");
 
 const teamService = new TeamService();
 
+const setOrganisation = async (req, res) => {
+  const { name } = req.body;
+  try {
+    if (!name || typeof name !== 'string' || name.trim().length === 0) {
+      return res.status(400).json({ error: 'Organisation name is required and must be a non-empty string' });
+    }
+
+    name = name.trim();
```

Biome

[error] Can't assign name because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)